tests/codebuild/run_all_sample_test.sh

```diff
@@ -15,6 +15,6 @@ verify_integration_tests ${crd_namespace}
 # run_update_integration_tests ${crd_namespace}
 # verify_update_integration_tests ${crd_namespace}
 verify_feature_integration_tests ${crd_namespace}
-run_smlogs_integration_tests ${crd_namespace}
+# run_smlogs_integration_tests ${crd_namespace}
```

Comment:
need to make the changes to the smlog so that the package compiles 

tests are removed coz we don't actually need smlog and i didn't resolve runtime errors

Reply:
rather than making changes to the package, deleting the tests, is it possible to comment out the compilation for sake of this PR?

basically, I am looking to commenting out everything (build, test, release) from the pipeline related to smlogs in one PR. right now its partial?

Reply:
fixed smlog